binancial/data/get_trades_realtime.py

```
@@ -1,45 +1,95 @@
             
-            f.write(trade + '\n')
-        
         except Exception as error:
-
-            print("MSG : " + str(msg))
-            print("Exception : " + str(error))
-
-            stream.stop_socket(stream)
-
-    stream.start_trade_socket(callback=handle_socket_message, symbol=symbol)
+            logger.error(f"Error processing message: {error}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)